core/jvm/src/main/scala/zio/internal/ZScheduler.scala

```diff
@@ -153,39 +164,26 @@ private final class ZScheduler(autoBlocking: Boolean) extends Executor {
             }
```

Comment:
Isn't this part always `false`? as we already checked `worker.blocking` before? 🤔
Sorry, it's not related to your work. I was just reading your PR while I stubbled upon this 🤔

Reply:
It's a volatile `var` so it _can_ be changed from another thread 🥲

Reply:
My bad! Thanks :)